apps/site/assets/ts/stop/components/DisplayTime.tsx

```diff
@@ -94,49 +77,34 @@ function BaseTimeDetails(): JSX.Element {
 }
 
 function DelayedTimeDetails(): JSX.Element {
-  const { departure, isLessThanHourAway, isCR, targetDate } = useContext(
-    DepartureContext
-  );
-  const { prediction, schedule } = departure;
-  const strikedthroughSchedule = (
-    <BasicTime
-      displayType="absolute"
-      time={schedule!.time}
-      targetDate={targetDate}
-      strikethrough
-    />
-  );
-  if (isLessThanHourAway && isCR) return strikedthroughSchedule;
-  if (!isLessThanHourAway) return strikedthroughSchedule;
+  const { departure, targetDate } = useContext(DepartureContext);
+  const scheduledTime = departure.schedule!.time;
   return (
     <>
-      {"Delayed "}
+      Delayed{" "}
       <BasicTime
         displayType="absolute"
-        time={prediction!.time}
+        time={scheduledTime}
         targetDate={targetDate}
-      />{" "}
-      {strikedthroughSchedule}
+        strikethrough
+      />
     </>
   );
 }
 
 const CancelledTimeDetails = (): JSX.Element => {
-  const { time, targetDate } = useContext(DepartureContext);
-
+  const { departure, targetDate } = useContext(DepartureContext);
```

Comment:
nitpick: can this and scheduledTime be defined globally for the file since it's also used for delayed times?

Reply:
this and that component were using the same React context, which was storing those variables. anyway, have refactored that away.